sway.el

```diff
@@ -501,6 +507,21 @@ argument for the undertaker.."
 ;;(let ((process-environment (frame-parameter frame 'environment)))
 ;;(call-process sway-swaymsg-binary nil nil nil (format sway-focus-message-format focused))))
 
+;;; Embark
```

Comment:
The origin of the approach is here https://github.com/melpa/melpa/pull/8350#issuecomment-1399330846 
I guess this would be the way to go if we add a lot of embark-related code, but for this simple first approach, I would better keep this as is. Also I think we shouldn't make a new package, rather an extension if the embark-related code becomes cumbersome enough to have it all in one file.

Reply:
if those few lines are all we need for Embark support, I guess it's not a big deal.